.github/workflows/package.yml

```diff
@@ -28,11 +28,21 @@ jobs:
               pyinstaller tabcmd-windows.spec --clean --noconfirm --distpath ./dist/windows
             OUT_FILE_NAME: tabcmd.exe
             ASSET_MIME: application/vnd.microsoft.portable-executable
+          - os: macos-13
```

Comment:
I noticed you used different os versions for mac architecture, have to tried seeing if you can generate the tabcmd app using the universal2 architecture so it works for both intel and arm? See --target-arch for universal2 https://pyinstaller.org/en/v5.13.0/man/pyi-makespec.html?highlight=target-arch#mac-os-specific-options

Reply:
good point, I'll give it a try

Reply:
only if it makes things simpler.

Reply:
I believe the github action `actions/setup-python` doesn't install universal2 python on our `mac-latest` os version run causing errors with the target_arch specified as `universal2`

Reply:
Okay that's fine, so long as we have a version of tabcmd for arm and intel we should be good.